cs/src/core/Index/FasterLog/FasterLog.cs

```diff
@@ -1034,8 +1034,8 @@ private bool ShouldCommmitMetadata(ref FasterLogRecoveryInfo info)
         
         private void CommitMetadataOnly(ref FasterLogRecoveryInfo info, bool spinWait)
         {
-            var fromAddress = CommittedUntilAddress > info.BeginAddress ? CommittedUntilAddress : info.BeginAddress;
-            var untilAddress = CommittedUntilAddress > info.BeginAddress ? CommittedUntilAddress : info.BeginAddress;
+            var fromAddress = FlushedUntilAddress > info.BeginAddress ? FlushedUntilAddress : info.BeginAddress;
```

Comment:
Should it be from CommittedUntil to FlushedUntil?

Reply:
yes, fixed.